compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.h

```diff
@@ -53,6 +55,11 @@ struct AMDAIEOptions {
         llvm::cl::cat(category),
         llvm::cl::desc("Path to MLIR-AIE installation directory"));
 
+    binder.opt<std::string>(
+        "iree-amd-aie-install-dir", amdAieInstallDir, llvm::cl::cat(category),
+        llvm::cl::desc("Path to AMDAIE installation directory (typically the "
+                       "IREE install directory)"));
+
```

Comment:
It would be nice if we didnt have to add yet another flag to our compile flag. Especially this one from the user point of view will look unnecessary as it would be the same path they are running iree-compile from. But I am guessing this is both necessary and temporary so its okay. Another option is that we can expect the user to add certain directories to their PATH and make it a part of our README then this flag wont be needed but you can do whichever you think is better since if this is temporary it doesnt matter much.   However, we need to advertise this change in a few places before we can land this.
1. A full example iree-compile command using peano in the description of this PR.
2. In our e2e wiki here https://github.com/nod-ai/iree-amd-aie/wiki/End-to-end-walk-through#e2e-compilation-command
3. Any internal resources where we show an e2e command, @daveliddell can help with that one.
As long as 1. and 2. are done we are good to land this PR in my opinion.

Reply:
> add certain directories to their PATH

I thought about this ofc and decided that this is the path to madness (that mlir-aie finds itself in with bash wrappers on wrappers on wrappers). And yes this flag is indeed temporary because the shell outs to `bootgen` and `amdaie_xclbinutil` will disappear.

Re updating the docs: this change is actually specifically/exactly to help @daveliddell put together e2e flows for Linux foudation dliverables. So I think he'll be able to tell me all the places that need to be updated.

Reply:
Yes, I will take responsibility for documenting the public dev build instructions and testing them.  I can also update the e2e wiki.  The playbook instructions will probably be in flux, as the public instructions should eventually replace them.  However, work remaining includes testing on Windows and coming up with more definitive machine (as opposed to workspace) setup instructions.

Reply:
All set - I'll hit `Save page` on the e2e docs as soon as this PR merges.